reference/html/flatpak-Error-codes.html

```diff
@@ -38,14 +38,24 @@ <h2><span class="refentrytitle"><a name="flatpak-Error-codes.top_of_page"></a>Er
 <col width="150px" class="functions_return">
 <col class="functions_name">
 </colgroup>
-<tbody><tr>
+<tbody>
+<tr>
 <td class="function_type">
-<span class="returnvalue">GQuark</span>
+<a href="/usr/share/gtk-doc/html/glib/glib-Quarks.html#GQuark"><span class="returnvalue">GQuark</span></a>
```

Comment:
Hmm I guess links like these should be suppressed or made to point to the GNOME docs website.